### PR TITLE
MAT-8058 Updated Springboot version also using freemarker provided by…

### PR DESCRIPTION
**Pull request by RohitKandimalla**
… springboot starter

## MADiE PR

Jira Ticket: [MAT-0000](https://jira.cms.gov/browse/MAT-0000)
(Optional) Related Tickets:

### All Submissions
* [ ] This PR has the JIRA linked.
* [ ] Required tests are included.
* [ ] No extemporaneous files are included (i.e Complied files or testing results).
* [ ] This PR is merging into the **correct branch**.
* [ ] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [ ] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

**Review by nmorasb — APPROVED** [review]

File: src/main/java/gov/cms/madie/services/PackagingService.java [hunk 1]
```diff
@@ -82,12 +82,18 @@ public byte[] createQRDA(QrdaRequestDTO qrdaRequestDTO, String accessToken) {
         qrdaExport.getSummaryReport().getBytes());
     for (QrdaReportDTO qrda : qrdaExport.getIndividualReports()) {
       if (qrda.getQrda() == null) {
-        log.warn("QRDA export for Measure [{}], testcase filename [{}] is missing QRDA", measure.getId(), qrda.getFilename());
+        log.warn(
```
Comment (adongare) [inline src/main/java/gov/cms/madie/services/PackagingService.java:85]: do we collect this logging level? I know we do info and error

**Review by adongare — APPROVED** [review]

File: src/main/java/gov/cms/madie/services/PackagingService.java (hunk 1 above)
Comment (RohitKandimalla) [reply src/main/java/gov/cms/madie/services/PackagingService.java:85]: Idk what log levels we are collecting, if we already not collecting warn, then I think we should, Because in this use case warn makes sense.

Comment (RohitKandimalla) [reply src/main/java/gov/cms/madie/services/PackagingService.java:85]: I can check with @sb-bdonohue
